Dockerfile

```
@@ -44,10 +44,19 @@ RUN apt-get update && \
 #ENV HTTP_PROXY=${http_proxy} \
 #    HTTPS_PROXY=${https_proxy} \
 #    NO_PROXY=${no_proxy}
-#COPY MY_ORG_ROOT.crt /usr/local/share/ca-certificates
-#RUN cd /usr/local/share/ca-certificates && \
-#    wget http://pki.my-organization.local/MY%20ORG%20ROOT.crt && \
-#    update-ca-certificates
+
+# Handle custom CA certificate, if specified.
+RUN if test -n "${TRAM_CA_URL}" -a -n "${TRAM_CA_THUMBPRINT}" ; then \
+        echo "Installing certificate authority from ${TRAM_CA_URL}" && \
+        curl -sk "${TRAM_CA_URL}" -o /usr/local/share/ca-certificates/tram_ca.crt && \
+        DOWNLOAD_CA_THUMBPRINT=$(openssl x509 -in /usr/local/share/ca-certificates/tram_ca.crt -fingerprint -noout | cut -d= -f2) && \
+        if test "${DOWNLOAD_CA_THUMBPRINT}" = "${TRAM_CA_THUMBPRINT}"; then \
+            update-ca-certificates; \
+        else \
+            echo "\n=====\nERROR\nExpected thumbprint: ${TRAM_CA_THUMBPRINT}\nActual thumbprint:   ${DOWNLOAD_CA_THUMBPRINT}\n=====\n"; \
```

```suggestion
            printf "\n=====\nERROR\nExpected thumbprint: ${TRAM_CA_THUMBPRINT}\nActual thumbprint:   ${DOWNLOAD_CA_THUMBPRINT}\n=====\n"; \
```

Based on GitHub Action `/github/workspace/Dockerfile:49 SC2028 info: echo may not expand escape sequences. Use printf.`